subsys/bluetooth/services/ias/ias_client.c

```diff
@@ -0,0 +1,147 @@
+/*
+ * Copyright (c) 2022 Codecoup
+ *
+ * SPDX-License-Identifier: Apache-2.0
+ */
+
+#include "bluetooth/att.h"
+#include "bluetooth/uuid.h"
+#include "net/buf.h"
+#include <errno.h>
+#include <stdint.h>
+#include <zephyr.h>
+#include <sys/byteorder.h>
+#include <sys/check.h>
+
+#include <device.h>
+#include <init.h>
+
+#include <bluetooth/bluetooth.h>
+#include <bluetooth/conn.h>
+#include <bluetooth/gatt.h>
+
+#include <bluetooth/services/ias.h>
+#include "../../host/conn_internal.h"
```

Comment:
No need to include internal header I guess

Reply:
Done